README.md

```diff
@@ -100,7 +100,7 @@ Example:
 	
 With word embeddings we can understand that vector operations describe word similarity. This means that we can see vector proofs of statements such as:
 
-	king - queen == man - woman
+	king-queen==man-woman
 
 
 #### Stop Words
```

Comment:
Could you please add description for section `Syntax dependecies`. 

Since PR is already open, can you also move these NLP term explanations and examples into a notebook directly as we discussed.